plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb

```diff
@@ -0,0 +1,49 @@
+module SamsonSlackWebhooks
+  class SlackWebhooksService
+    # users in the format jquery mentions input needs
+    # see _notify_buddy_box.html.erb
+    def users
+      Rails.cache.fetch(:slack_users, expires_in: 5.minutes, race_condition_ttl: 5) do
+        if slack_api_token
```

Comment:
can we early return instead of nesting the rest of the function body in a conditional? Asking because it is quite deeply nested